all/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(build_all LANGUAGES CXX)
+
+# Missing dependency?
+# add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/../bench" "${CMAKE_BINARY_DIR}/bench")
```

Comment:
I have no experience with Google Benchmark and what files it generates.

If you mean how to pull it in as a dependency without installing to the system, starting with CMake 3.14, you just do the following:
```cmake
include(FetchContent)
FetchContent_Declare(benchmark
        GIT_REPOSITORY ...
        GIT_TAG ...
        UPDATE_DISCONNECTED YES)
FetchContent_MakeAvailable(benchmark)
```
This will handle things for you, you will now just have to `target_link_libraries(... PRIVATE ...)` whatever targets you need.